pkg/operator/controller/names.go

```diff
@@ -0,0 +1,9 @@
+package nodeobservabilitycontroller
+
+const (
+	OperatorNamespace      = "node-observability-operator"
+	AgentName              = "node-observability-agent"
+	AgentImage             = "quay.io/node-observability-operator/node-observability-agent:latest"
+	KubeletCAConfigMapName = "kubelet-serving-ca"
+	KubeletSourceNamespace = "openshift-config-managed"
```

Comment:
These variables are not used anywhere, they have to be added in the other PR about the kubelet configmap.